src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/prevalue/umb-block-grid-area-editor.html

```diff
@@ -2,33 +2,39 @@
 
     <umb-load-indicator ng-if="vm.loading"></umb-load-indicator>
 
-    <div ng-show="vm.loading !== true" class="umb-block-grid-area-editor__grid-wrapper" style="--umb-block-grid--block-grid-columns: {{vm.block.areaGridColumns || vm.rootLayoutColumns}}">
+    <div ng-if="vm.loading !== true" 
```

Comment:
Consider inverting this as the value of `loading` might be whatever.

```suggestion
<div ng-if="!vm.loading"
```